supersuit/vector/sb3_vector_wrapper.py

```diff
@@ -14,7 +14,9 @@ def reset(self, seed=None, options=None):
         return self.venv.reset()
 
     def step_wait(self):
-        return self.venv.step_wait()
+        obss, rews, terms, truncs, infos = self.venv.step_wait()
```

Comment:
This change wasn't intentional, I think it must have been due to using an older version of master. I'm going to try removing it and seeing if the tests pass still

Reply:
Just removed it, will check back if it passed later

Reply:
CI passed but we don't have CI testing it with SB3, and looking at the docs they are keeping the old gym v21 API with done instead of truncations/terminations, so I'm going to revert this